test/box/gh-5799-fiber-info.result

```diff
@@ -0,0 +1,73 @@
+-- test-run result file version 2
+test_run = require('test_run').new()
+ | ---
+ | ...
+fiber = require('fiber')
+ | ---
+ | ...
+json = require('json')
+ | ---
+ | ...
+
+function yielder(n) for i=1, n do fiber.yield() end end
+ | ---
+ | ...
+
+csw_check_counter = 0
+ | ---
+ | ...
+fibers = {}
+ | ---
+ | ...
+
+test_run:cmd('setopt delimiter ";"')
+ | ---
+ | - true
+ | ...
+for i=1,100 do
+    fibers[i] = fiber.new(function()
+        for j=1,10 do
+            fiber.yield()
+            if j == fiber.self():csw() and j == fiber.self():info().csw then
```

Comment:
Please, keep the lines in 80 symbols length limit. This is clearly longer.

Reply:
Fixed